monitoring/uss_qualifier/scenarios/astm/netrid/dss_wrapper.py

```diff
@@ -460,7 +461,7 @@ def _fail_sub_check(
         isa_validator = ISAValidator(
             main_check=main_check,
             scenario=self._scenario,
-            isa_params=None,  # won't check the ISA's content
+            isa_params=expected_isa_params,  # won't check the ISA's content
```

Comment:
The comment is not relevant anymore, is it?
```suggestion
            isa_params=expected_isa_params,
```